.install/rocky_linux_9.sh

```diff
@@ -4,8 +4,8 @@ set -e
 export DEBIAN_FRONTEND=noninteractive
 $MODE dnf update -y
 
-$MODE dnf install -y gcc-toolset-13-gcc gcc-toolset-13-gcc-c++ make wget git openssl openssl-devel python3 which \
-    rsync unzip
+$MODE dnf install -y gcc-toolset-13-gcc gcc-toolset-13-gcc-c++ make wget git \
+    openssl openssl-devel python3 python3-devel which rsync unzip clang which
```

Comment:
`which` is listed twice
```suggestion
    openssl openssl-devel python3 python3-devel which rsync unzip clang
```